compose.yaml

```diff
@@ -1,3 +1,6 @@
+version: '3.8'
```

Comment:
Regarding the [Compose Sepcification](https://github.com/compose-spec/compose-spec/blob/master/spec.md) the `version` has been deprecated and thus should no longer be used.

> The Compose file is a YAML file defining version (DEPRECATED), services (REQUIRED), [...]